Registry/registry.solar_fields

```diff
@@ -28,12 +28,13 @@ state   real    tau_qi        ij     dyn_em       1         -     h      "TAU_QI
 state   real    tau_qs        ij     dyn_em       1         -     h      "TAU_QS"       "MASS-WEIGHTED SNOW OPTICAL THICKNESS"                       " "
 state   real    tau_qc_tot    ij     dyn_em       1         -     h      "TAU_QC_TOT"   "MASS-WEIGHTED LIQUID CLOUD OPTICAL THICKNESS RES + UNRES"   " "
 state   real    tau_qi_tot    ij     dyn_em       1         -     h      "TAU_QI_TOT"   "MASS-WEIGHTED ICE OPTICAL THICKNESS RES + UNRES"            " "
-state   real    cbaseht       ij     dyn_em       1         -     h      "CBASEHT"      "CLOUD BASE HEIGHT"                                          "m"
-state   real    ctopht        ij     dyn_em       1         -     h      "CTOPHT"       "CLOUD TOP HEIGHT"                                           "m"
-state   real    cbaseht_tot   ij     dyn_em       1         -     h      "CBASEHT_TOT"  "CLOUD BASE HEIGHT RES + UNRES"                              "m"
-state   real    ctopht_tot    ij     dyn_em       1         -     h      "CTOPHT_TOT"   "CLOUD TOP HEIGHT RES + UNRES"                               "m"
+state   real    cbaseht       ij     dyn_em       1         -     h      "CBASEHT"      "CLOUD BASE HEIGHT"                                          "m agl"
+state   real    ctopht        ij     dyn_em       1         -     h      "CTOPHT"       "CLOUD TOP HEIGHT"                                           "m agl"
+state   real    cbaseht_tot   ij     dyn_em       1         -     h      "CBASEHT_TOT"  "CLOUD BASE HEIGHT RES + UNRES"                              "m agl"
+state   real    ctopht_tot    ij     dyn_em       1         -     h      "CTOPHT_TOT"   "CLOUD TOP HEIGHT RES + UNRES"                               "m agl"
```

Comment:
@twjuliano What does 'RES' and 'UNRES' mean here?

Reply:
@weiwangncar 'RES' and 'UNRES' mean resolved and unresolved. I can explicitly write that out if you'd like. Thanks.

Reply:
@twjuliano Thanks. I think spelling them out would be helpful.